curriculum/challenges/english/20-upcoming-python/learn-list-comprehension-by-building-a-case-converter-program/657ed53c19461d4b95c4757a.md

```diff
@@ -0,0 +1,56 @@
+---
+id: 657ed53c19461d4b95c4757a
+title: Step 2
+challengeType: 20
+dashedName: step-2
+---
+
+# --description--
+
+Now create a new list inside the function named `snake_cased_char_list`. You can use a set of empty square braces or the built-in `list()` method to create the new list. This list will hold the characters after you have converted them to snake case.
+
+# --hints--
+
+You should name the list `snake_cased_char_list`.
+
+```js
+({
+  test: () => {
+    assert(__pyodide.runPython(`
+      import inspect
+      a = __locals.get('convert_to_snake_case')
+      src = inspect.getsource(a)
+      'snake_cased_char_list' in src
+    `));
+  }
+})
+```
+
+You should use a set of empty square braces or the built-in `list()` function to create the list.
+
+```js
+({
+  test: () => {
+    assert(__pyodide.runPython(`
+      import inspect
+      a = __locals.get('convert_to_snake_case')
+      src = inspect.getsource(a)
+      'snake_cased_char_list = []' in src
+      or
+      'snake_cased_char_list = list()' in src
+    `));
+  }
+})
+```
+
+# --seed--
+
+## --seed-contents--
+
+```py
+def convert_to_snake_case(pascal_or_camel_cased_string):
+--fcc-editable-region--
+    
+--fcc-editable-region--
+    pass
```

Comment:
```suggestion
--fcc-editable-region--
def convert_to_snake_case(pascal_or_camel_cased_string):
    pass    
--fcc-editable-region--
```
Do not make the editable region too narrow. Also, ask/remember the user to use `pass` when required. It's fine if you do it in the hint